Resources/Prototypes/Recipes/Construction/Graphs/structures/fence_metal.yml

```diff
@@ -9,31 +9,27 @@
     - to: straight
       completed:
       - !type:SnapToGrid
```

Comment:
```suggestion
```

Since you removed the southRotation datafield from `SnapToGrid`, and it had no other purpose, you can remove the SnapToGrid action itself, as well. And then, since there is nothing else left on the `completed` datafield, you can remove that line too.
Then do the same with every other construction step you have removed southRotation from

Reply:
Does this anchor it or is the prototype itself anchored? It might be essential

Reply:
It's anchored